lib/iex/test/iex/helpers_test.exs

```diff
@@ -71,6 +71,7 @@ defmodule IEx.HelpersTest do
     cleanup_modules([Sample])
   end
 
+  @tag :does_not_apply_to_otp19
```

Comment:
The only issue is that on OTP 19 the module will be in memory and thus we cannot get the abstract code in the h helper.